src/components/StaticPages/About.css

```diff
@@ -0,0 +1,9 @@
+.center{
+	text-align: center;
+}
+.logo-link{
+	display: inline-block;
+}
+.logo{
```

Comment:
This style also affected to other logo image.
![image](https://user-images.githubusercontent.com/12796262/77733423-7107ba00-7039-11ea-91bc-5b4874c56aa3.png)